website/siteConfig.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = [
+  {
+    caption: 'User1',
+    // You will need to prepend the image path with your baseUrl
+    // if it is not '/', like: '/test-site/img/image.jpg'.
+    image: '/img/delphix-logo-white.png',
+    infoLink: 'https://www.delphix.com',
+    pinned: true,
+  },
+];
+
+const siteConfig = {
+  title: 'Couchbase Plugin', // Title for your website.
+  tagline: 'Virtualise couchbase data sets',
+  url: 'https://delphix.github.io', // Your website URL
+  baseUrl: '/couchbase-plugin/', // Base URL for your project */
+  projectName: 'couchbase-plugin',
+  organizationName: 'delphix',
+
+
+
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    {doc: 'Overview', label: 'Docs'},
+    {page: 'help', label: 'Help'},
+    { search: true },
+  ],
+
+  // If you have users set above, you add it here:
+
+
+  /* path to images for header/footer */
+  headerIcon: '/img/delphix-logo-white.png',
```

Comment:
For some reason the delphix logo looks stretched in the header (https://pratap-akhand.github.io/couchbase-plugin/docs/Overview).

Reply:
Fixed